server/data/data-library/src/main/java/org/apache/james/domainlist/lib/AbstractDomainList.java

```diff
@@ -87,6 +87,10 @@ public void configure(DomainListConfiguration domainListConfiguration) throws Co
         addConfiguredDomains(domainListConfiguration.getConfiguredDomains());
     }
 
+    public void configure(DomainListConfiguration.Builder configurationBuilder) throws ConfigurationException {
```

Comment:
belongs to previous commit, right ?

Reply:
When you do such comment, can you include current commit SHA please? It makes finding the incriminated commit easier.

Reply:
I reworked the history.